docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+# docker-compose up --env-file=.env.docker
```

Comment:
Please note that to test the compose, you need to inject correct env file. The default one woudn't work as it's shared with React as well.